spec/rester/client/response_spec.rb

```diff
@@ -1,11 +1,14 @@
 module Rester
   class Client
     RSpec.describe Response do
-      let(:response) { Response.new(status) }
+      let(:response) { Response.new(status, params) }
       let(:status) { 200 }
-      subject { response }
+      let(:params) { {} }
 
-      it { is_expected.to be_a Hash }
+      describe '#to_h' do
+        subject { response.to_h }
+        it { is_expected.to be_a Hash }
+      end # #to_h
 
       describe '#initalize' do
         def frozen?(object)
```

Comment:
This method seems broken.  I don't think it's actually running any checks because it's checking if the object responds to `:frozen` instead of `:frozen?`.